examples/pruning/chainermn_integration.py

```diff
@@ -99,15 +99,19 @@ def objective(trial, comm):
     evaluator = chainermn.create_multi_node_evaluator(evaluator, comm)
     report = evaluator()
 
-    return 1.0 - report['main/accuracy']
+    return report['main/accuracy']
 
 
 if __name__ == '__main__':
     # Please make sure common study and storage are shared among nodes.
     study_name = sys.argv[1]
     storage_url = sys.argv[2]
 
-    study = optuna.load_study(study_name, storage_url, pruner=optuna.pruners.MedianPruner())
+    study = optuna.load_study(
+        study_name,
+        storage_url,
+        pruner=optuna.pruners.MedianPruner(),
+        direction='maximize')
```

Comment:
Please remove `direction='maximize'` because `optuna.load_study` does not have `direction` argument.
In addition to that, please add `--direction maximize` to [L11](https://github.com/pfnet/optuna/pull/526/files#diff-2d1b3387c842d67fa53c28335838bf46R11) as follows:

```
    $ STUDY_NAME=`optuna create-study --storage $STORAGE_URL --direction maximize`
```